tftest.py

```diff
@@ -0,0 +1,14 @@
+import gensim,os,time
+from gensim.models.tfword2vec import TfWord2Vec
+from gensim.models.word2vec import Word2Vec, Text8Corpus
+
+#TF benchmark
+start = time.time()
```

Comment:
Is this file needed? I will be adding the same code in the notebook. I think this file should be removed.